lib/optcarrot/nes.rb

```diff
@@ -89,5 +92,11 @@ def run
     ensure
       dispose
     end
+
+    private
+
+    def save_fps_history?
+      @conf.print_fps || @conf.print_p95fps
```

Comment:
Please replace `@conf.print_fps` with `@conf.print_fps_history`

Reply:
Oops, that was a mistake. Thanks for catching that!